### PR TITLE
LINK-1962 | Input field length validation

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Validate maximum length of input fields

## Closes
[LINK-1962](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1962)

[LINK-1962]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1962?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr148-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/domain/signupGroup/__tests__/validation.test.ts [hunk 1]
```diff
@@ -234,6 +235,46 @@ describe('signupSchema function', () => {
...
 
+  const testCases: [Partial<SignupFormFields>][] = [
+    [{ city: '' }],
+    [{ city: mockString(51) }],
+    [{ firstName: '' }],
+    [{ firstName: mockString(51) }],
+    [{ lastName: '' }],
+    [{ lastName: mockString(51) }],
+    [{ phoneNumber: '' }],
+    [{ phoneNumber: 'xxx' }],
```
Comment (harriris-vincit) [inline src/domain/signupGroup/__tests__/validation.test.ts:246]: A max length test case would be a nice addition

File: src/domain/signupGroup/__tests__/validation.test.ts [hunk 2]
```diff
@@ -417,43 +367,29 @@ describe('getContactPersonSchema function', () => {
...
-      await testContactPersonSchema(registration, signups, {
-        ...validContactPerson,
-        phoneNumber: 'xxx',
-        notifications: [NOTIFICATIONS.SMS],
-      })
-    ).toBe(false);
-  });
+  const testCases: [Partial<ContactPersonFormFields>][] = [
+    [{ email: '' }],
+    [{ email: 'not-email' }],
```
Comment (harriris-vincit) [inline src/domain/signupGroup/__tests__/validation.test.ts:372]: Here too max length test cases for email and phone number would be fitting additions

**Review by harriris-vincit — APPROVED** [review]
Two suggestions regarding test cases, but otherwise looks very good.

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr148-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr148-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/signupGroup/__tests__/validation.test.ts (hunk 2 above)
Comment (jorilindell) [reply src/domain/signupGroup/__tests__/validation.test.ts:374]: Added test cases for phone number and email max lengths

**Pull request merged**